Client/Frontend/Settings/LoginDetailViewController.swift

```diff
@@ -154,15 +162,20 @@ extension LoginDetailViewController: UITableViewDataSource {
             loginCell.descriptionLabel.text = login.hostname
             websiteField = loginCell.descriptionLabel
             websiteField?.accessibilityIdentifier = "websiteField"
+            loginCell.isEditingFieldData = isEditingFieldData
             return loginCell
 
         case .lastModifiedSeparator:
-            let cell = ThemedTableViewCell(style: .subtitle, reuseIdentifier: nil)
-            let lastModified = NSLocalizedString("Last modified %@", tableName: "LoginManager", comment: "Footer label describing when the current login was last modified with the timestamp as the parameter.")
-            let t = UInt64(login.timePasswordChanged)
-            let formattedLabel = String(format: lastModified, Date.fromTimestamp(t).toRelativeTimeString())
+            let cell = CenteredDetailCell(style: .subtitle, reuseIdentifier: nil)
+            let created = NSLocalizedString("Created %@", tableName: "LoginManager", comment: "Label describing when the current login was created with the timestamp as the parameter.")
```

Comment:
The existing NSLocalizedString was here, so I followed suit. Perhaps all the NSLocalizedStrings in this file should be pulled out for consistency.

Reply:
a good idea. but for now its best to leave this here because its pretty late in the l10n cycle.